spec/js/views/app_view_spec.js

```diff
@@ -1,32 +1,38 @@
 define([
```

Comment:
Incomplete tests, is order to make sure the functionality was implemented there should be a tests reflecting all cases:

- empty input
- all spaces input
- valid input